src/pe_reports/report_generator.py

```diff
@@ -521,18 +527,17 @@ def main():
     logging.info("Generating Graphs")
 
     # Create output directory
-    subprocess.call(["mkdir", args["OUTPUT_DIRECTORY"]])
+    os.mkdir(f"{args['OUTPUT_DIRECTORY']}")
```

Comment:
try:
   os.mkdir(f"{args['OUTPUT_DIRECTORY']}")
except FileExistsError as err:
    logging.error(f"The output directory exists {err}", exc_info=True)
        return 0